### PR TITLE
Switch to chrome for selenium tests

### PR DESCRIPTION
**Pull request by vanzaj**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,28 @@
+sudo: required
```
Comment (basvodde) [inline .travis.yml]: This one should be very uncomfortable as then it can't run fast, if I'm right

Comment (vanzaj) [inline .travis.yml]: doesn't seem to affect speed that much...

**Comment by basvodde** [comment]
It should in start-up. See: https://docs.travis-ci.com/user/ci-environment/

File: .travis.yml [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
   - gem update
   - gem install bundler
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
-  - sudo pip install robotframework
-  - sudo pip install robotframework-selenium2library
-  - npm install geckodriver
+  - pip install robotframework
+  - pip install robotframework-selenium2library
+  - ls robottests/ChromeProfiles/zh-cn/Default/
```
Comment (basvodde) [inline .travis.yml]: Why the ls here? If this is debug, should it be moved up?

File: .travis.yml [hunk 3]
```diff
@@ -1,18 +1,29 @@
...
+
 
 before_script:
```
Comment (basvodde) [inline .travis.yml]: If empty, remove?

File: .travis.yml [hunk 4]
```diff
@@ -1,18 +1,29 @@
...
-  - firefox --version
-  - echo $DISPLAY
 
 rvm:
   - 2.1
```
Comment (basvodde) [inline .travis.yml]: For future, upgrade ruby?

File: robottests/language_detection.txt [hunk 5]
```diff
@@ -8,7 +8,8 @@ Detect Default Language (English) Of Browser
     Page Language Should Be English
 
 Detect (Simplified) Chinese Language Of Browser
-    Open Browser  ${BASE_URL}  ${BROWSER}  ff_profile_dir=robottests/FirefoxProfiles/zh-cn
+    #Open Browser  ${BASE_URL}  ${BROWSER}  ff_profile_dir=robottests/FirefoxProfiles/zh-cn
```
Comment (basvodde) [inline robottests/language_detection.txt]: Commented out line. Delete? Also delete the FirefoxProfiles dir?

File: robottests/language_detection.txt [hunk 6]
```diff
@@ -17,4 +18,17 @@ Page Language Should Be English
...
+Open Chrome With Zh Locale
+    [Arguments]     ${url}
+    [Documentation]     Starts the Google Chrome browser.
+    ${options}=    Evaluate    sys.modules['selenium.webdriver'].ChromeOptions()    sys, selenium.webdriver
+    Call Method     ${options}     add_argument     --user-data-dir\=robottests/ChromeProfiles/zh-cn
+    #Call Method     ${options}     add_argument     --user-data-dir\=/dev/null
+    #Call Method     ${options}     add_argument     --incognito
+    #Call Method     ${options}     add_argument     --profile-directory\=robottests/ChromeProfiles/zh-cn/Default
+    #Call Method     ${options}     add_argument     --bwsi
+    #Call Method     ${options}     add_argument     --managed-user-id\=""
```
Comment (basvodde) [inline robottests/language_detection.txt]: Lots of commented out lines?

File: robottests/resource.txt [hunk 7]
```diff
@@ -3,4 +3,4 @@ Library  Selenium2Library
 
 *** Variables ***
 ${BASE_URL}      http://localhost:9292/
-${BROWSER}       firefox
+${BROWSER}       gc
```
Comment (basvodde) [inline robottests/resource.txt]: Unused variable?

File: robottests/translation_switcher.txt [hunk 8]
```diff
@@ -17,11 +17,11 @@ Check if it is possible to switch to Japanese language for the page
 *** Keywords ***
 
 Open the About Page and switch to  [Arguments]  ${language}
-	Open Browser  ${BASE_URL}/about
-	Click Link    ${language} 
+    Open Browser  ${BASE_URL}/about  ${BROWSER}
```
Comment (basvodde) [inline robottests/translation_switcher.txt]: Use Open Chrome here too?

File: robottests/check_website_links.txt [hunk 9]
```diff
@@ -1,7 +1,7 @@
 *** Settings ***
 Resource      resource.txt
 
-Test Setup    Open Browser    ${BASE_URL}
+Test Setup    Open Browser    ${BASE_URL}    ${BROWSER}
```
Comment (basvodde) [inline robottests/check_website_links.txt]: Open Chrome ?

File: robottests/language_detection.txt [hunk 10]
```diff
@@ -8,7 +8,8 @@ Detect Default Language (English) Of Browser
     Page Language Should Be English
 
 Detect (Simplified) Chinese Language Of Browser
-    Open Browser  ${BASE_URL}  ${BROWSER}  ff_profile_dir=robottests/FirefoxProfiles/zh-cn
+    #Open Browser  ${BASE_URL}  ${BROWSER}  ff_profile_dir=robottests/FirefoxProfiles/zh-cn
+    Open Chrome With Zh Locale  ${BASE_URL}
```
Comment (basvodde) [inline robottests/language_detection.txt]: Open Browser

File: robottests/resource.txt (hunk 7 above)
Comment (basvodde) [inline robottests/resource.txt]: Can be ignored. Is used.

File: robottests/translation_switcher.txt (hunk 8 above)
Comment (basvodde) [inline robottests/translation_switcher.txt]: Can be ignored.

File: robottests/language_detection.txt (hunk 5 above)
Comment (basvodde) [inline robottests/language_detection.txt]: Still commented out line?

File: .travis.yml [hunk 11]
```diff
@@ -1,18 +1,29 @@
+sudo: false
```
Comment (vanzaj) [inline .travis.yml]: Without `sudo: false` line, robot tests fail to complete due to timeouts. Tests still run in new containers.

Comment (basvodde) [inline .travis.yml]: Any idea why?

**Comment by basvodde** [comment]
I'll merge this so we can look at the next change independently.

**Pull request merged**